lib/topics_filter.rb

```diff
@@ -417,7 +418,7 @@ def exclude_topics_with_any_tags(tag_ids)
 
   def include_topics_with_all_tags(tag_ids)
     tag_ids.each do |tag_id|
-      sql_alias = "tt#{topic_tags_alias}"
+      sql_alias = topic_tags_alias
```

Comment:
The `tt` prefix was unnecessary because `topic_tags_alias` already includes the `tt` prefix.